feat: hide `install steam` for owned games

nolddor opened the pull request:
closes #223 

Sample URL: https://store.steampowered.com/app/252950/Rocket_League/

**BEFORE**
![image](https://github.com/user-attachments/assets/1388f6b8-7fe1-4bee-9878-fe5f3a9c8115)

**AFTER**
![image](https://github.com/user-attachments/assets/72729448-a87b-400e-9764-89e8070d78cc)

xPaw commented:
I don't like this being in global. Would be better to check this option in the app script instead.

And I think you can directly hide `.game_area_already_owned_btn`?

nolddor commented:
Hi @xPaw `.game_area_already_owned_btn` is shared with the `Play Now` and applied to the `div` container not the `a` regretfully, that's why I added `:first-of-type`.

Regarding the location, I'm not aware of the plugin structure just checked where the global button was removed and added the extra code. Will be sending a second commit to allocate the logic in the right place.

Thanks for the quick reply and review.

xPaw commented:
![image](https://github.com/user-attachments/assets/03c8c5de-e5bc-4404-b786-12e72c9b4cb9)
is it?

xPaw commented:
Ok that is annoying, if they change the order of buttons or something, that can cause issues.

nolddor commented:
Yes,.. there is no a good/right way to do this. I don't see why they did not add their own `class` or `id`.. The href attribute are different tho.

`Play now` is `href="steam://launch/252950/Dialog`
`Install Steam` is `href="https://store.steampowered.com/about/?snr=1_5_9__owned-game`

Any suggestion how to do the filtering are welcome, I do not see any good solution to be fair.

xPaw commented:
I think select first of type link, and check that URL(href).pathname is "/about/" before hiding it

nolddor commented:
Moved logic from `global.js` to `app.js`

xPaw commented:
Looks good, can you move it after the ifs() with Draw calls though.

nolddor commented:
Sure, no issues. Now should be in the right order.
